src/main/java/com/github/dockerjava/api/model/Container.java

```diff
@@ -83,6 +89,12 @@
     @JsonProperty("NetworkSettings")
     private ContainerNetworkSettings networkSettings;
 
+    /**
+     * @since ~{@link RemoteApiVersion#VERSION_1_23}
+     */
+    @JsonProperty("Mounts")
+    private ContainerMount[] mounts;
```

Comment:
Please use List or Set

Reply:
Sure. I could change this.
BTW, for `public ContainerPort[] ports`, do we also want to use List or Set for it?

Reply:
old code not changed, will be changed in future